src/Umbraco.Web.UI.Client/src/common/security/retryqueue.js

```diff
@@ -23,13 +25,18 @@ angular.module('umbraco.security.retryQueue', [])
         }
       });
     },
-    pushRetryFn: function(reason, retryFn) {
+    pushRetryFn: function(reason, userName, retryFn) {
       // The reason parameter is optional
-      if ( arguments.length === 1) {
-        retryFn = reason;
+      if ( arguments.length === 2) {
+        retryFn = userName;
+        userName = reason;

```

Comment:
don't think so. we're shifting args because reason is missing.
previously: retryFn <- reason <- null
now: retryFn <- userName <- reason <- null
think it works?